frontend/app/(dashboardLayout)/equity/dividend_rounds/page.tsx

```
@@ -30,17 +31,20 @@ const columns = [
   const router = useRouter();
+  const user = useCurrentUser();
+  const pathname = usePathname();
   const [dividendRounds] = trpc.dividendRounds.list.useSuspenseQuery({ companyId: company.id });
-
   const table = useTable({ columns, data: dividendRounds });
+  const currentLink = navLinks(user, company).find((link) => link.route === pathname);
 
   return (
-    <EquityLayout>
+    <>
+      {currentLink && <PageHeader currentLink={currentLink} />}
```

ESLint

[error] Potential leaked value that might cause unintentionally rendered values or rendering crashes

(react/jsx-no-leaked-render)